### DISCUSSION TITLE
Where are?

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [jscad/OpenJSCAD.org#785] [General]
Thanks for opening up github discussions as a first step to resolve #783 . This move will be useful.

**Where are ... ?**
* The alternative sites for www.openresearch.org to work around #779?
* The sources for V1 / V2
* The documentation pages for V1/V2 - current and deprecated ones
* The chat rooms of the past?

**Comment by WolfgangFahl** [comment]
Alternative Site:
V1:
- https://openjscad.surge.sh/
V2:
- http://www.gfnews.net/jscad/

Sources:
- https://github.com/jscad/OpenJSCAD.org/tree/V1
- https://github.com/jscad/OpenJSCAD.org/tree/V2

but V2 README seems to be inconsistent.


Documentation:
I see e.g. https://github.com/jscad/OpenJSCAD.org/blob/V1/docs/index.md but there seem to be no V2 equivalent
V2:
- http://www.gfnews.net/jscad/docs/index.html

**Comment by z3dev** [comment]
And for those that don't know, here's the JSCAD User Group; https://openjscad.nodebb.com/
